### PR TITLE
Rewrote experimental Layout Page

### PR DESCRIPTION
**Pull request by jeoffw**
References #25 

Major rewrite of the experimental Layout page to capture key concepts in applying new grid.

#### Changelog

**New**

- Added content on fluid, fixed, and hybrid grid, with discussion of how to choose.

**Changed**

- Consolidated sizing/spacing guidance into Sizing Scale section.

**Removed**

- Deleted some redundant content.

### CODE REVIEW

File: src/content/experimental/layout/layout.md [hunk 1]
```diff
@@ -1,24 +1,283 @@
...
+A [fluid grid](#fluid-grid) divides space by two. This division is immediately evident in the 2x column structure. Within a breakpoint, column count is constant and unit size scales with screen size. Choose a column count by starting with one, then divide by two as needed:
+
+| Fluid Grid Columns
+| -------------------
+| 1
+| 2
+| 4
+| 8
+| 16
+
```
Comment (alisonjoseph) [inline src/content/experimental/layout/layout.md]: Did you intend the above to be a table? Its currently displaying like this 
screen shot 2018-09-26 at 8 46 31 am

**Comment by jeoffw** [comment]
Yeah that was supposed to be a table. It previewed correctly in vscode, so I wonder what I did wrong.

**Comment by alisonjoseph** [comment]
Interesting, looks like it was just missing the end `|` marks. I can fix it!

**Comment by alisonjoseph** [comment]
@jeoffw would you prefer if we remove the width: 100% from the tables so they don't stretch across the page? See the difference below. 

screen shot 2018-09-26 at 9 17 34 am
screen shot 2018-09-26 at 9 17 28 am

**Pull request merged**